kowl/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.1.0
+version: 2.2.0
```

Comment:
Can you remove this, so that I can package another change into this release? Changing this will automatically trigger the release process otherwise

Reply:
No problem